language/stdlib/modules/Libra.move

```diff
@@ -572,14 +572,18 @@ module Libra {
     /// `MintCapability<CoinType>` and `BurnCapability<CoinType>` resources.
     public fun register_currency<CoinType>(
         account: &signer,
-        _: &Capability<RegisterNewCurrency>,
+        // DD refactor
+        // _: &Capability<RegisterNewCurrency>,
+        tc_account: &signer,
         to_lbr_exchange_rate: FixedPoint32,
         is_synthetic: bool,
         scaling_factor: u64,
         fractional_part: u64,
         currency_code: vector<u8>,
     ): (MintCapability<CoinType>, BurnCapability<CoinType>)
     acquires CurrencyRegistrationCapability {
+        // Privilege check.  TODO (dd): Error code.
```

Comment:
I think we should also introduce `assert_has_register_new_currency_privilege` and then have the error code produced for this there once and for all.

Reply:
Yeah, no more thinking up error codes sounds nice.

Reply:
Ok.  I think I'll forge ahead with the current approach, but plan to change it after I've finished refactoring.  This sounds like a change that would be easy to do with sed.

Reply:
I changed it so that each function has assert(has_..._privilege, <code>) in each function call, instead of the previous assert_has_...._privilege with one common error code.  Error codes are unique but not sensibly assigned (Tim said he'll reassign them).